src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((value) => value.trim())
+    .filter((value) => value !== '')
+    .reduce((prev, curr) => {
+      return {
+        ...prev,
+        [curr.split(':')[0].trim()]: curr.split(':')[1].trim(),
```

Comment:
[CODE KNOWLEDGE] - You're calling `curr.split(':')` twice, which is unnecessary. This operation can be done once and the results stored in variables to be used in the object construction. This will improve performance, as you won't be splitting the string twice for every style declaration.